app/views/govuk_publishing_components/components/feedback/_problem_form.html.erb

```diff
@@ -46,3 +52,17 @@
     </div>
   </div>
 </form>
+
+<script>
```

Comment:
What's the reason for inlining the js rather than adding this to the existing js for the component?
https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/javascripts/govuk_publishing_components/components/feedback.js

Reply:
I've added it inline in case of a scenario where a bot is able to parse the page, without downloading any of the external scripts. This way seemed to be a more reliable way to make sure the script is executed (but otherwise i'd definitely prefer to include it in the external script).

Reply:
Seems like a good reason! Could you put a comment to that effect above it for future devs who wonder?

Reply:
👍 updated